config/default/test/contacts.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     },
 
     'type': 'person',
+    'created_by_doc': 'fake_delivery_report_uuid',
```

Comment:
Using this in the [`pnc_danger_sign_baby` task tests](https://github.com/medic/cht-core/pull/7739/files#diff-0931f83e17f49e76f2ced633d5bdcf8fc8b63f4cf98c7c2da900706e86d7c609) to verify that `__delivery_uuid` field is properly set from the task inputs.